nginx/files/proxy.conf

```diff
@@ -5,28 +5,37 @@ server {
   {%- include "nginx/files/_name.conf" %}
   {%- include "nginx/files/_ssl.conf" %}
 
-  {%- if site.get('ssl', {'enabled': False}).get('enabled', False) %}
-  ssl_session_cache shared:SSL:10m;
-  {%- endif %}
-
   {%- if site.get('underscores_in_headers', False) %}
   underscores_in_headers on;
   {%- endif %}
 
   {%- include "nginx/files/_auth.conf" %}
 
-  location / {
-      {%- if site.proxy.upstream_proxy_pass is defined %}
-      proxy_pass {{ site.proxy.upstream_proxy_pass }};
+  {%- set location = {} %}
+  {%- if site.get('location', False) %}
```

Comment:
FYI: .get defaults to something evaluated as false so you don't need to explicitly use False as a default value